src/main/java/uk/gov/justice/digital/job/DomainRefreshJob.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.justice.digital.job;
+
+import io.micronaut.configuration.picocli.PicocliRunner;
+import org.apache.spark.SparkConf;
+import org.apache.spark.sql.SparkSession;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+import uk.gov.justice.digital.client.dynamodb.DynamoDBClient;
+import uk.gov.justice.digital.config.JobParameters;
+import uk.gov.justice.digital.repository.DomainRepository;
+import uk.gov.justice.digital.service.DomainRefreshService;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+/**
+ * Job that refreshes domains so that the data in the consumer-facing systems is correctly formatted and up-to-date.
+ *  It reads domains from DomainRegistry (a DynamoDB table) and does one of the following:
+ *   1. Refresh whole domain
+ *   2. Refresh one table in a domain
+ */
+@Singleton
+@CommandLine.Command(name = "DomainRefreshJob")
```

Comment:
Minor - you can declare a static import for the `Command` annotation like the `DataHubJob`.